client/PRUserial485/PRUserial485.py

```diff
@@ -14,62 +14,34 @@
 - 0: command
 - 1..4: payload length
 - 5.. : payload
-"""
 
-# # It is a python module. It must not be run itself.
-# if (__name__ == "__main__"):
-#     exit()
+Note: library clients should first
+       a) invoke function "PRUserial_set_bbb_ip_address" and
+       b) invoke function "PRUserial485_threads_start",
+      before using communication functions.
+"""
 
 import socket
-import sys
 import struct
-import time
 from threading import Thread, Event
 from queue import Queue
 from PRUserial485 import constants_PRUserial485_bridge as _c
-from PRUserial485.functions_PRUserial485_bridge import find_BBB_IP \
-    as _find_BBB_IP
-from siriuspy import util as _util
 
 
 SERVER_PORT_RW = 5000
 SERVER_PORT_GENERAL = 6000
-BBB_NAME = ''
-BBB_IP = ''
-
-
-# Queues
-general_queue = Queue()
-rw_queue = Queue()
-
-# Dictionary for answers
-answers = {}
 
 
-
-def set_beaglebone_ip(bbbname=None, ip=None):
-    """."""
-    global BBB_NAME
-    global BBB_IP
-    if ip is None:
-        if bbbname is None:
-            bbbname = socket.gethostname()
-
-        BBB_NAME = bbbname.replace('--', ':')
-        BBB_IP = _find_BBB_IP(BBB_NAME)
-        if BBB_IP == '':
-            sys.stdout.write(_util.get_timestamp() +
-                             ": Beaglebone IP not found. Please check BBB NAME\n")
-            sys.stdout.flush()
-            sys.exit()
-            sys.stdout.write(_util.get_timestamp() + ": '" + BBB_NAME + "'" +
-                             " will be connected on " + BBB_IP + "\n")
-            sys.stdout.flush()
-    else:
-        BBB_IP = ip
+# BeagleBone IP
+BBB_IP = None
 
 
+# Queues
+QUEUE_GENERAL = Queue()
+QUEUE_RW = Queue()
 
+# Dictionary for COMM_RESPONSE
+COMM_RESPONSE = {}
```

Comment:
Any specific reason for renaming these variables?

Reply:
We use visual studio code editor which has an audit software called pylama. Pylama suggests using capital letters for module variables, according to PEP008, https://www.python.org/dev/peps/pep-0008/

But this is just a recommendation that avoids visual studio marking issues along the code. should we keep the original names?